src/main/resources/templates/files.html

```diff
@@ -15,7 +15,20 @@
     </nav>
 </header>
 <main th:fragment="bodycontent">
-    <!-- Restlichen Inhalt hier einfügen! -->
+    <table>
+        <tr>
+            <th>Name</th>
+            <th>Datum</th>
+            <th>Typ</th>
+            <th>Größe</th>
+        </tr>
+        <tr th:each="file : ${files}" th:href="@{/${file.id}}">
+            <td th:text="${file.fileName}">%Name%</td>
+            <td th:text="${file.date}">%Datum%</td>
+            <td th:text="${file.contentType}">%Typ%</td>
+            <td th:text="${file.size}">%Größe%</td>
+        </tr>
```

Comment:
Is it possible to refactor it to fragments, so we could use it in `directory` as well, or would that be over kill?

Reply:
Added as issue #110